core/string/translation.cpp

```diff
@@ -1095,10 +1095,18 @@ StringName TranslationServer::translate(const StringName &p_message, const Strin
 	}
 
 	if (!res) {
-		return p_message;
+		if (pseudolocalization_enabled) {
+			return pseudolocalize(p_message);
+		} else {
+			return p_message;
+		}
```

Comment:
You could use a ternary operator here:
```suggestion
		return pseudolocalization_enabled ? pseudolocalize(p_message) : p_message;
```
Same below.